connectors/src/connectors/slack/temporal/activities.ts

```diff
@@ -20,7 +20,7 @@ const { NANGO_SLACK_CONNECTOR_ID } = process.env;
 const logger = mainLogger.child({ provider: "slack" });
 
 // This controls the maximum number of concurrent calls to syncThread and syncNonThreaded.
-const MAX_CONCURRENCY_LEVEL = 2;
+const MAX_CONCURRENCY_LEVEL = 10;
```

Comment:
:fire: move :)

Reply:
put it back to 2. I tested locally, it was hitting the rate limiter too often.